db/seeds.rb

```diff
@@ -8,3 +8,4 @@
 require File.dirname(__FILE__) + '/seed_helper.rb'
 
 create_roles_and_permissions
+populate_languages
```

Comment:
So currently this 'populate_languages' method is defined only within the seed helper, rather than a standalone rake task. Can somewhere else call this method from the seed helper? If so where should it be called from?